io.github.maurycyliebner.enve.yaml

```diff
@@ -0,0 +1,186 @@
+    - cp -r * /app/gperftools_root
+  cleanup: [/gperftools_root]
+  sources:
+    - type: archive
+      url: https://github.com/gperftools/gperftools/releases/download/gperftools-2.7/gperftools-2.7.tar.gz
+      sha256: 1ee8c8699a0eff6b6a203e59b43330536b22bbcbe6448f54c7091e5efb0763c9
+    - type: patch
+      path: gperftools-enve-mod.patch
+
+- shared-modules/python2.7/python-2.7.json
+
+- name: gn
+  buildsystem: simple
+  build-commands:
+    - python2 build/gen.py
+    - ninja -C out
+    - install -D out/gn -t /app/bin
+  cleanup: ["*"]
+  sources:
+    - type: git
+      disable-shallow-clone: true
+      url: https://gn.googlesource.com/gn
+      commit: 4cd7fe1509f6e8b7a60430c6c19f2f03159d7889
+
+- name: skia
+  buildsystem: simple
+  build-commands:
+    - gn gen out/Release --args='is_official_build=true
+                                 is_debug=false
+                                 is_component_build=false
+                                 target_os="linux"
+                                 target_cpu="x64"
+                                 skia_use_dng_sdk=false
+                                 skia_use_system_libjpeg_turbo=false
+                                 skia_use_libwebp_decode=false
+                                 skia_use_libwebp_encode=false
+                                 skia_use_zlib=true'
+    - ninja -C out/Release skia
+    - mkdir /app/skia_root
+    - cp -r * /app/skia_root
+  cleanup: ["*"]
+  sources:
+    - type: git
+      dest: third_party/externals/sfntly
+      url: https://github.com/googlefonts/sfntly.git
+      commit: 1e7adf313bd9c488a70015f8df8782f7c65e9ce7
+    - type: archive
+      dest: third_party/externals/libjpeg-turbo
+      url: https://downloads.sourceforge.net/project/libjpeg-turbo/2.0.4/libjpeg-turbo-2.0.4.tar.gz
+      sha256: 33dd8547efd5543639e890efbf2ef52d5a21df81faf41bb940657af916a23406
+    - type: git
+      # url: https://github.com/google/skia.git
+      url: https://skia.googlesource.com/skia
+      commit: 76f4fff613b5ba4365d24dbd943a8ed47221f1e9
+
+- name: enve
+  buildsystem: simple
+  build-options:
+    env:
+      QMAKEPATH: /app/lib
+  build-commands:
+    - mkdir _build && cd _build &&
+      qmake -r .. CONFIG+=release
+                  CONFIG+=force_debug_info
+                  INCLUDEPATH+=/app/include/QtWebEngineWidgets
+                  INCLUDEPATH+=/app/skia_root
+                  INCLUDEPATH+=/app/gperftools_root/src
+                  LIBS+=-L/app/lib
+                  LIBS+=-L/app/skia_root/out/Release
+                  LIBS+=-L/app/gperftools_root &&
+      sed -e 's|/usr/lib/x86_64-linux-gnu/libQt5WebEngine|/app/lib/x86_64-linux-gnu/libQt5WebEngine|g'
+          -i src/app/Makefile &&
+      make -j $FLATPAK_BUILDER_N_JOBS &&
+      cp -v src/app/enve /app/bin &&
+      cp -v src/core/libenvecore.so.1 /app/lib
+    - install -Dm644 build/AppDir/usr/share/icons/hicolor/48x48/enve.png -t /app/share/icons/hicolor/48x48/apps
```

Comment:
Would be nice to get these renamed upstream and installed under correct name here until it happens.

Reply:
I know, I'm already working on it (have already some commits upstream).